src/rgw/rgw_rados.cc

```diff
@@ -1492,7 +1492,7 @@ int RGWZoneParams::fix_pool_names()
   }
 
   domain_root = fix_zone_pool_name(pool_names, name, ".rgw.data.root", domain_root.name);
-  metadata_heap = fix_zone_pool_name(pool_names, name, ".rgw.meta", metadata_heap.name);
+  metadata_heap = "";
```

Comment:
sorry, my bad, I couldn't get around testing that with vstart, that said, with the above suggested change, I guess we would need to default to empty string, isn't it?